INSTALL.win.rst

```diff
@@ -0,0 +1,129 @@
+=======
+Windows
+=======
+
+
+  Tested on Windows 8.1 and Windows 10 1809 (both amd64)
+
+
+Build Requirements
+------------------
+
+- `Build Tools for Visual Studio 2019 <https://visualstudio.microsoft.com/downloads>`_
+- `cmake <https://cmake.org/download>`_
+- `ninja <https://github.com/ninja-build/ninja/releases>`_
+- `git <(https://git-scm.com/download/win>`_
+- `nasm <https://www.nasm.us>`_
+- python (3.7+)
+- pip
+
+**Ensure that every requirement is on the system path and Windows can find it:**
+
+.. code-block:: sh
+
+  > where cmake
+  C:\Program Files\CMake\bin\cmake.exe
+  > where ninja
+  C:\Windows\System32\ninja.exe
+  > where git
+  C:\Program Files\Git\cmd\git.exe
+  C:\Program Files\Git\mingw64\bin\git.exe
+  > where nasm
+  C:\Program Files\NASM\nasm.exe
+  > where python
+  C:\Python39\python.exe
+  > where pip
+  C:\Python39\Scripts\pip.exe
+
+Your paths can be different but if you get ``INFO: Could not find files for the given pattern(s).`` that means this executable is not in path.
+
+
+Build Steps
+-----------
+
+1. Open the Native Tools Command Prompt for VS 2019
+
+   It will open command line window but from it you will be able to compile code.
+
+
+2. Install cffi module from pypi:
+
+   - ``pip install --upgrade cffi``
```

Comment:
Same here on CFFI, this shouldn't require manual installs and comments from the Linux file.